cypress/e2e/requests.cy.js

```diff
@@ -0,0 +1,22 @@
+describe("Requests Page", () => {
+  it("A logged out user should not able to see the requests list.", () => {
+		// Visit a protected route in order to allow cypress to set the cookie and mock the login
+		cy.visit("/requests")
+
+		cy.get('div.alert-heading').contains('Unauthorized').then(() => {
+			cy.log("A logged out user is not able to view requests.");
+		});
+	})
+  
+	it("A logged in user should be able to see the requests list.", () => {
+		// Call the custom cypress command to log in
+		cy.login(Cypress.env('TEST_SCIENTIST_USER'), Cypress.env('TEST_SCIENTIST_PW'))
+
+		// Visit a protected route in order to allow cypress to set the cookie and mock the login
+		cy.visit("/requests")
+
+		cy.get('h1').contains('My Requests').then(() => {
+			cy.log("Successfully logged in and viewing requests page");
+		});
+	})
+});
```

Comment:
now that the wcl has been updated to display text when we have no requests, can that test be added too?

Reply:
i was going to do that in a separate PR because i needed to figure out how and it was going to take a while. Since the test looks at the db itself and I already have requests, i also will not be able to check if its working.

Reply:
nvm, went ahead and included a check for this, but this is just checking that either of these elements exist when the user is signed in vs. saying that it needs to know whether or not a user has requests on their request list or not. 

i think if we wanted to do the latter we would need to test the data itself, which isn't done with cypress out of the box but theres an add on called `cypress-react-selector` where we could potentially be able to do something like this. 
heres an example of how it could be used: https://glebbahmutov.com/blog/react-state-from-e2e-tests/

but this is more in depth and should probably be a separate ticket

Reply:
is mock data not an option?

the current implementation is ok for now, but it should be expanded in the future. although the tests says it's checking for the existence of requests or a "no requests" message....it isn't. since we don't have a use case where tests exist, the "no requests" route is the only one that will ever return true.

Reply:
there is a way with cypress `fixtures` - if we did that and only wanted to use mock data, we would need to mock every single api call- this is an example of that: https://blog.bitsrc.io/mocking-http-calls-in-cypress-end-to-end-tests-fa2e6b7caaf7

it can be done but it isn't really in the scope of this ticket